src/index.html

```diff
@@ -11,12 +11,84 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Kalnia+Glaze&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="home"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo"
+        />
+      </a>
+      <nav class="nav">
+        <a
+          href="apple"
+          class="nav_link is-active"
+        >
+          Apple
+        </a>
+        <a
+          href="samsung"
+          class="nav_link"
+        >
+          Samsung
+        </a>
+        <a
+          href="smartphones"
+          class="nav_link"
+        >
+          Smartphones
+        </a>
+        <a
+          href="laptops & computers"
+          data-qa="hover"
+        >
+          Laptops & Computers
+        </a>
+        <a
+          href="gadgets"
+          class="nav_link"
+        >
+          Gadgets
+        </a>
+        <a
+          href="tablets"
+          class="nav_link"
+        >
+          Tablets
+        </a>
+        <a
+          href="photo"
+          class="nav_link"
+        >
+          Photo
+        </a>
+        <a
+          href="video"
+          class="nav_link"
+        >
+          Video
+        </a>
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. For instance, `class="nav_link is-active"` can be renamed to `class="nav__link nav__link--active"` to follow BEM naming conventions.